client/keys/add.go

```diff
@@ -75,6 +77,24 @@ the flag --nosort is set.
 	return cmd
 }
 
+func getKeybase(cmd *cobra.Command, dryrun bool, buf io.Reader) (keys.Keybase, error) {
```

Comment:
The argument `cmd` does not seem to be used, please remove it:

```suggestion
func getKeybase(dryrun bool, buf io.Reader) (keys.Keybase, error) {
```

Reply:
oops, was previously used, forgot to remove